test/integration/default/controls/_mapdata_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# Replace per minion strings
+replacement = {
+  hostname: system.hostname
+}
+
+mapdata_file = "_mapdata/#{system.platform[:finger].split('.').first}.yaml"
+mapdata_dump = inspec.profile.file(mapdata_file) % replacement
```

Comment:
Maybe add a line to explain what's going here : what does `inspec.profile.file(mapdata_file)` return?

Reply:
This is an InSpec feature to [access files stored as resources in the current profile](https://docs.chef.io/inspec/profiles/#profile-files).

I can add a comment with the URL to the documentation.

Reply:
I wonder if `inputs` https://docs.chef.io/inspec/inputs/ could also be used simply (without us doing the job to find the correct file).

Reply:
@daks I'm not sure it will make things easier :thinking:

Reply:
> I can add a comment with the URL to the documentation.

yes please :+1:

Reply:
@baby-gnu not sure neither, was just an idea, I have no problem with the current implementation